examples/storm-starter/src/jvm/storm/starter/BlobStoreAPIWordCountTopology.java

```diff
@@ -0,0 +1,248 @@
+import backtype.storm.generated.AuthorizationException;
+import backtype.storm.generated.InvalidTopologyException;
+import backtype.storm.generated.KeyAlreadyExistsException;
+import backtype.storm.generated.KeyNotFoundException;
+import backtype.storm.generated.SettableBlobMeta;
+import backtype.storm.spout.SpoutOutputCollector;
+import backtype.storm.task.ShellBolt;
+import backtype.storm.task.TopologyContext;
+import backtype.storm.topology.BasicOutputCollector;
+import backtype.storm.topology.IRichBolt;
+import backtype.storm.topology.OutputFieldsDeclarer;
+import backtype.storm.topology.TopologyBuilder;
+import backtype.storm.topology.base.BaseBasicBolt;
+import backtype.storm.topology.base.BaseRichSpout;
+import backtype.storm.tuple.Fields;
+import backtype.storm.tuple.Tuple;
+import backtype.storm.tuple.Values;
+import backtype.storm.utils.Utils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Random;
+
+public class BlobStoreAPIWordCountTopology {
+    private static NimbusBlobStore store = new NimbusBlobStore(); // Client API to invoke blob store API functionality
+    private static String key = "key1";
+    private static final Logger LOG = LoggerFactory.getLogger(BlobStoreAPIWordCountTopology.class);
+    private static final int READ = 0x01;
+    private static final int WRITE = 0x02;
+    private static final int ADMIN = 0x04;
+    private static final List<AccessControl> WORLD_EVERYTHING =
+            Arrays.asList(new AccessControl(AccessControlType.OTHER, READ | WRITE | ADMIN));
+
+    // Spout implementation
+    public static class RandomSentenceSpout extends BaseRichSpout {
+        SpoutOutputCollector _collector;
+        BlobStoreAPIWordCountTopology wc;
+        String key;
+        NimbusBlobStore store;
+
+
+        @Override
+        public void open(Map conf, TopologyContext context, SpoutOutputCollector collector) {
+            _collector = collector;
+            wc = new BlobStoreAPIWordCountTopology();
+            key = "key1";
+            store = new NimbusBlobStore();
+            store.prepare(Utils.readStormConfig());
+        }
+
+        @Override
+        public void nextTuple() {
+            Utils.sleep(100);
+            try {
+                 _collector.emit(new Values(wc.getBlobContent(key, store)));
+            } catch (AuthorizationException | KeyNotFoundException | IOException exp) {
+                throw new RuntimeException(exp);
+            }
+        }
+
+        @Override
+        public void ack(Object id) {
+        }
+
+        @Override
+        public void fail(Object id) {
+        }
+
+        @Override
+        public void declareOutputFields(OutputFieldsDeclarer declarer) {
+            declarer.declare(new Fields("word"));
```

Comment:
I guess you meant to say the spout part to be @Override
+        public void declareOutputFields(OutputFieldsDeclarer declarer) {
+            declarer.declare(new Fields("sentence"));
+        } rather than bolt which splits the sentence. Made the changes thanks for the catch